tests/common.go

```diff
@@ -13369,3 +13369,31 @@ func DeleteFilesFromS3Bucket(bucketName string, fileName string) error {
 	log.Infof("The files %v are successfully deleted from the bucket [%s]", keys, bucket)
 	return nil
 }
+func GetPvcFromNamespace(namespace string, pvclist []string) ([]string, error) {
+	var pvclist []string
+	allPvcList, err := core.Instance().GetPersistentVolumeClaims(namespace, nil)
+	if err != nil {
+		log.InfoD("error getting pvcs from namespace [%s]", namespace)
+		return nil, err
+	}
+
+	for _, p := range allPvcList.Items {
+		pvclist = append(pvclist, p.Name)
+		return pvclist, nil
+	}
+	return nil, nil
+}
+
+func GetVolumeNamefromPVC(namespace string) ([]string, error) {
```

Comment:
logic in this function and above function is same

Reply:
removed it